src/Converter/Feature2Mesh.js

```diff
@@ -65,8 +65,12 @@ class FeatureMesh extends THREE.Group {
                 // calculate the scale transformation to transform the feature.extent
                 // to feature.extent.as(crs)
                 coord.crs = Crs.formatToEPSG(this.#originalCrs);
-                extent.copy(this.extent).applyMatrix4(this.#collection.matrix);
-                extent.as(coord.crs, extent);
+                if (this.extent.isExtent) {
+                    extent.copy(this.extent).applyMatrix4(this.#collection.matrix);
+                    extent.as(coord.crs, extent);
+                } else {
+                    this.extent.toExtent(coord.crs, extent);
```

Comment:
Shouldn't it always be an Extent now? In which case is it a Tile?

Reply:
This is one of the pain point with this PR, it seems that pre-PR the `Extent` is tiled one fore Mapbox Vector Tiles and a geographic one for WFS...

I choose not to fix it for now, hoping that migrating to typescript will help us here (at least tracking the types and incompatible one).

But if you think we should fix it here, I will need some more time and help. x)

Reply:
I'm ok with fixing it later but I think we should write it somewhere to not forget about it (either in the ts migration proposal or in a new issue, or in a todo)